wrapper.go

```diff
@@ -43,3 +43,10 @@ type Wrapper interface {
 	// the underlying wrapper. Supported options: WithAad.
 	Decrypt(ctx context.Context, ciphertext *BlobInfo, options ...Option) ([]byte, error)
 }
+
+// KeyBytes defines an optional interface for wrappers to implement that returns
+// the "current" key bytes
+type KeyBytes interface {
```

Comment:
I assume the idea here is that it's wrapper-specific what these bytes are? Cause in some cases it could be something like an asymmetric key mapping to big ints rather than e.g. a byte slice -- but it could be marshaled in some standardized way (pkcs<X> for instance).